.eslintrc.js

```diff
@@ -6,5 +6,4 @@
  */
 module.exports = {
   extends: ['eslint-config-salesforce-typescript', 'eslint-config-salesforce-license'],
-  rules: { 'prettier/prettier': ['error', { endOfLine: 'auto' }] },
```

Comment:
this baked in now?

Reply:
prettier is no longer an eslint plugin.  It runs based on its own rules.  There's a prettier config that removes all the eslint rules that conflict with prettier so that there's only one ruleset in charge.

see https://github.com/forcedotcom/eslint-config-salesforce/commit/fe3ccd41936a5b57a68a60fa1fcf889f77bd6408